### PR TITLE
update hatchery for qa IBD

### PR DESCRIPTION
**Pull request by wangfan860**
Link to Jira ticket if there is one: https://ctds-planx.atlassian.net/browse/PXP-8340

### Description of changes

- make a `hatchery` folder and `hatchery.json` to store the hatchery settings.
- delete the hatchery settings in `manifest.json`

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
# qa-ibd.planx-pla.net
 :x: etlMapping.yaml - gitops.json mismatch
- Field error: Field [project_id] in explorerConfig.filters.tabs.fields not found in etlMapping
- Field error: Field [project_id] in explorerConfig.table.fields not found in etlMapping
- Field error: Field [project_id] in explorerConfig.table.fields not found in etlMapping

**Comment by PlanXCyborg** [comment]
# qa-ibd.planx-pla.net/manifest.json
## :warning: Services on branch
- arborist
- dashboard
- fence
- indexd
- peregrine
- pidgin
- revproxy
- sheepdog
- portal
- spark
- tube
- hatchery
- wts
- manifestservice
- guppy
- ssjdispatcher
- metadata
- sower

File: qa-ibd.planx-pla.net/manifests/hatchery/hatchery.json [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        "/bin/sh",
+        "root"
+      ]
+    },
+    "containers": [
+      {
+        "target-port": 8787,
+        "cpu-limit": "1.0",
+        "memory-limit": "1024Mi",
+        "name": "R Studio 4.1.0",
```
Comment (paulineribeyre) [inline qa-ibd.planx-pla.net/manifests/hatchery/hatchery.json:32]:
```suggestion
        "name": "RStudio 4.1.0",
```

File: qa-ibd.planx-pla.net/manifests/hatchery/hatchery.json [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        "use-tls": "false",
+        "ready-probe": "/",
+        "user-volume-location": "/home/rstudio/pd",
+        "fs-gid": 100
+      },
+      {
+        "target-port": 8787,
+        "cpu-limit": "1.0",
+        "memory-limit": "1024Mi",
+        "name": "R Studio 4.1.0 with packages",
```
Comment (paulineribeyre) [inline qa-ibd.planx-pla.net/manifests/hatchery/hatchery.json:48]:
```suggestion
        "name": "RStudio 4.1.0 + base packages",
```

File: qa-ibd.planx-pla.net/manifests/hatchery/hatchery.json [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+        "ready-probe": "/",
+        "user-volume-location": "/home/rstudio/pd",
+        "fs-gid": 100
+      },
+      {
+        "target-port": 8888,
+        "cpu-limit": "1.0",
+        "memory-limit": "1024Mi",
+        "name": "Jupyter - Python/R",
+        "image": "quay.io/occ_data/jupyternotebook:1.7.2",
```
Comment (paulineribeyre) [inline qa-ibd.planx-pla.net/manifests/hatchery/hatchery.json:65]:
```suggestion
        "image": "quay.io/cdis/jupyter-notebook:1.1.0",
```

**Comment by PlanXCyborg** [comment]
# qa-ibd.planx-pla.net
 :x: etlMapping.yaml - gitops.json mismatch
- Field error: Field [project_id] in explorerConfig.filters.tabs.fields not found in etlMapping
- Field error: Field [project_id] in explorerConfig.table.fields not found in etlMapping
- Field error: Field [project_id] in explorerConfig.table.fields not found in etlMapping

**Comment by PlanXCyborg** [comment]
# qa-ibd.planx-pla.net/manifest.json
## :warning: Services on branch
- arborist
- dashboard
- fence
- indexd
- peregrine
- pidgin
- revproxy
- sheepdog
- portal
- spark
- tube
- hatchery
- wts
- manifestservice
- guppy
- ssjdispatcher
- metadata
- sower

**Comment by PlanXCyborg** [comment]
# qa-ibd.planx-pla.net
 :x: etlMapping.yaml - gitops.json mismatch
- Field error: Field [project_id] in explorerConfig.filters.tabs.fields not found in etlMapping
- Field error: Field [project_id] in explorerConfig.table.fields not found in etlMapping
- Field error: Field [project_id] in explorerConfig.table.fields not found in etlMapping

**Comment by PlanXCyborg** [comment]
# qa-ibd.planx-pla.net/manifest.json
## :warning: Services on branch
- arborist
- dashboard
- fence
- indexd
- peregrine
- pidgin
- revproxy
- sheepdog
- portal
- spark
- tube
- hatchery
- wts
- manifestservice
- guppy
- ssjdispatcher
- metadata
- sower

**Comment by PlanXCyborg** [comment]
# qa-ibd.planx-pla.net
 :x: etlMapping.yaml - gitops.json mismatch
- Field error: Field [project_id] in explorerConfig.filters.tabs.fields not found in etlMapping
- Field error: Field [project_id] in explorerConfig.table.fields not found in etlMapping
- Field error: Field [project_id] in explorerConfig.table.fields not found in etlMapping

**Comment by PlanXCyborg** [comment]
# qa-ibd.planx-pla.net/manifest.json
## :warning: Services on branch
- arborist
- dashboard
- fence
- indexd
- peregrine
- pidgin
- revproxy
- sheepdog
- portal
- spark
- tube
- hatchery
- wts
- manifestservice
- guppy
- ssjdispatcher
- metadata
- sower

**Review by paulineribeyre — APPROVED** [review]

**Pull request merged**